.github/workflows/renovate.yml

```diff
@@ -0,0 +1,31 @@
+# Renovate: https://github.com/renovatebot/github-action
```

Comment:
All our files in KubeArchive should have the Copyright and license identifier.

```suggestion
# Copyright KubeArchive Authors
# SPDX-License-Identifier: Apache-2.0
# Renovate: https://github.com/renovatebot/github-action
```

Reply:
fixed